static/js/base.js

```diff
@@ -64,14 +64,23 @@ function update(state) {
     Example of state: {x:72, y:72, dir:0}
     */
     const scaleFactor = 3;
-    const scaledX = state.X * scaleFactor - 30;
-    const scaledY = state.Y * scaleFactor - 40;
+    const centerX = state.X * scaleFactor;
+    const centerY = state.Y * scaleFactor;
     const dir = state.dir;
     document.getElementById("demo").innerHTML = state.X.toFixed(2) + ", " + state.Y.toFixed(2)
+    const sensorPoints = document.querySelectorAll("circle")
+    const scaledX = centerX - 30
+    const scaledY = centerY - 40
```

Comment:
I know scaledX and scaledY are the names we were using before, but something like topLeftX and topLeftY may make more sense.